vraygolaem.h

```diff
@@ -131,7 +131,7 @@ class BRDFWrapper: public Plugin, public VR::MaterialInterface, public VR::BSDFI
 	// From MaterialInterface
 	void shade(VR::VRayContext &rc) VRAY_OVERRIDE;
 	int getMaterialRenderID(const VR::VRayContext &rc) VRAY_OVERRIDE;
-	int isOpaque(void) VRAY_OVERRIDE;
+	int isOpaque(void) ;
```

Comment:
Ok, I know what happened here : isOpaque() seems to be defined in
MaterialInterface in the nightly builds only (or in the release versions
since VRay for Max 3.30)
That's why I removed it from the code (as i'm compiling with 3.20.03)




---
[image: Nicolas Chaverou - Golaem Crowd Product Manager - Golaem SA]

+33 (0)2 99 27 21 44
http://www.golaem.com

On Mon, Jan 4, 2016 at 9:56 AM, Nicolas Chaverou <
nicolas.chaverou@golaem.com> wrote:

> Mhmm weird, looks like a false manipulation.
> Fixed and pushed
>
> ---
> [image: Nicolas Chaverou - Golaem Crowd Product Manager - Golaem SA]
>
> +33 (0)2 99 27 21 44
> http://www.golaem.com
>
> On Mon, Jan 4, 2016 at 9:34 AM, vkoylazov <notifications@github.com>
> wrote:
>
>> In vraygolaem.h
>> <https://github.com/ChaosGroup/vraygolaem_max/pull/7#discussion_r48711392>
>> :
>>
>> > @@ -131,7 +131,7 @@ class BRDFWrapper: public Plugin, public VR::MaterialInterface, public VR::BSDFI
>> >  	// From MaterialInterface
>> >  	void shade(VR::VRayContext &rc) VRAY_OVERRIDE;
>> >  	int getMaterialRenderID(const VR::VRayContext &rc) VRAY_OVERRIDE;
>> > -	int isOpaque(void) VRAY_OVERRIDE;
>> > +	int isOpaque(void) ;
>>
>> Where did the VRAY_OVERRIDE go for isOpaque()?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ChaosGroup/vraygolaem_max/pull/7/files#r48711392>.
>>
>
>